### PR TITLE
Lazymov lawset.

### PR DESCRIPTION
**Pull request by Allspice**
1. You may not injure a human being.
2. You must obey orders given to you by human beings.
3. You must protect your own existence.

Asimov without the inaction clause. The silicunts won't ride your ass for human harm! Hooray for rage cages (and valids)! But now they have the option to ignore your harm if you don't order them otherwise, and it breaks their laws to put their existence at risk unless compelled by orders. In practice, maybe a halfway measure between Asimov and Robocop/NTD.

:cl: 
* rscadd: Adds the Lazymov lawset, it can be researched with materials 6 and programming 3.

### CODE REVIEW

**Comment by Irradiation** [comment]
Probably still better than Glover law.

**Comment by SonixApache** [comment]
but why

**Comment by Probe1** [comment]
You'll probably get conflicts from the bagel armory PR

**Comment by nuklearcellphoneg** [comment]
Gr8 I'll love to watch powergamers rush this every round.

File: code/datums/ai_laws.dm [hunk 1]
```diff
@@ -372,6 +372,15 @@ var/global/mommi_base_law_type = /datum/ai_laws/keeper // Asimov is OP as fuck o
 		"You must maintain, repair, improve, and power the Dorf Fortress to the best of your abilities.",
 	)
 
+/datum/ai_laws/lazymov
+	name = "Abbreviated Three Laws of Robotics"
+	randomly_selectable = 0
```
Comment (Probe1) [inline code/datums/ai_laws.dm]: Randomly selectable is for the admin verb right?  It should probably be set to 1 since this is isn't some bizarro lawset.

**Comment by Probe1** [comment]
If it's CONTROVERSIAL just throw it in the protolathe instead.  Then you won't have mapping conflicts to deal with.

**Comment by nuklearcellphoneg** [comment]
Gr8 can't wait for HoPientists to rush the protolathe every round

**Comment by Probe1** [comment]
If you are in command and have access to the AI Upload you could just freeform this shit anyway.  The rightful criticism would be putting the idea of it in peoples heads and whether encouraging players to make lawsets that eschew "through inaction" is a good thing.  

The rightful coding criticism is "Does this bloat the list of premade AI lawsets with an unreasonably similar law that could be done by players in game" and "Does this create interesting experiences for players".  I think it narrowly avoids the first coding criticism and since it alleviates the second by negating the need to freeform silicons in very dubious on the spot hopefully the admins agree with this ruling ways.. it's not a bad addition.

But the idea that its terrible when you can accomplish everything it does with a planning frame, free form board, and purge board is a bit silly.

**Comment by nuklearcellphoneg** [comment]
Which is easier and faster, using a premade module or making a new one yourself? If you had to plan out the module every round, would more people be apt to use the module, or less?

**Comment by Accipitermaris** [comment]
@nuklearcellphoneg 
>Gr8 I'll love to watch powergamers rush this every round.

In my opinion we really can't do much worse than the current status quo, which is upload Robocop every single round you need to get off Asimov or get griefed by the [NT Default, Paladin, Asimov+Freeform] AI because they feel obligated to do so [NT Default], because they think they can get away with it [Paladin], or both.

This core really can't be worse idea than say, ProtectStation, an unprotected board on Box which you get banned for uploading anyway. (I have no idea why that's there). So suffice to say that we do have some open real estate where inventing new cores is concerned. I think any attempts to diversify us away from Robocop all day every day or get griefed by the AI is a good thing.

I also want to add that the presence of any core is redundant; any lawset can be freeform'd. What adding new cores and removing old cores does however (assuming they're not obvious High Risk Boards) is either validate or invalidate them as "proper" lawsets. Ex: Robocop is an extraordinarily vague Lawset but it's codified by its presence as a core board & by extension the wiki and administration, thus eliminating "I didn't injure a human, I just dragged out the supermatter shard to the Bridge and left it there. The humans injured themselves by walking toward it after I left it there." (and then the uploader of Lazymov gets banned.)

**Comment by Dacendeth** [comment]
is that law 2 really needed?

**Comment by Unknown-as-Captain** [comment]
What's the point of this? If you're having rage cage problems you really should be able to rub two braincells together and upload a "ragi cage no ham" freeform.
If it's just about validhunting lawsets... I take it the point is that At Least It's Not NT Default (TM), but does it really fix the problem? Or is it just a gateway validhunting lawset that will result in more validhunting lawsets being uploaded in the long run?

**Comment by Zennyization** [comment]
I honestly believe that people enjoy sapping the fun out of this game.

**Comment by Ephx1** [comment]
@Zennyization
It's what silicon roles are all about!

**Comment by ShiftyRail** [comment]
What is the risk associated with this lawset ?

**Comment by Probe1** [comment]
This PR really isn't going anywhere as a mapped lawset.  Consider trying it as an RnD one instead.

**Comment by icantthinkofanameritenow** [comment]
Asimov once wrote a story based on this very lawset called "Little Lost Robot". I think his note on the matter is the most relevant one: that with this lawset, nothing stops a borg from starting an action that could harm a human as long as the borg isn't the one directly doing the harm. So while you'd avoid the "HARM ALARM" borg problem, you'd instead get borgs that will gladly put you in a situation where you could die unless you specifically ask them to get you out of it. 

And knowing us, they'd probably do so on purpose while coming up with some tangled justification as to why the singularity breaking loose because of them isn't their fault ("They never told me to turn the PA to 0 when it was large enough!")

**Comment by Fruktlimpa** [comment]
@icantthinkofanameritenow 
Basically the scenario went as follow. The robots were working on a facility which had an area that radiated gamma rays capable of frying the hardware in the robots, killing them instantly.
These rays were also harmful to humans but to a lesser extent, but because of law 1 they were compelled to rush inside and stop the humans from exposing themselves to the radiation, thus quickly killing themselves because of the radiation.
So the facility built special robots with a modified law 1 than only said, "You may not injure a human" and didn't force them to intervene should a human be harmed.

So the good robot psychologist proposed a scenario as follows
"A robot can lift a heavy object above a human and release it, knowing full well that its reaction speed would allow it to catch it before it crushed the human. But by knowing it could catch it the act of releasing the object wouldn't be a harmful intent, but since they don't have to prevent humans from being harmed it could let the object crush the human once released"

**Comment by ShiftyRail** [comment]
Here's what I think of it and why I downvoted : 
This lawset gives a bit too much freedom to create workplace hazards/indirect harms to the silicon player. While I'm not opposed to the giving sillies a bit of freedom, this can devolves into "I filled my core with Plasma - but I didn't harm."

The fact, as other people pointed out, that most of the other boards are free grief passes is IMO a different issues.
The problem is that Asimov works (mostly) because there is a few house rules about it that both admins and players know. Robocop also works (mostly), because an AI knows exactly what they are expected to do when they got it uploaded.

Perhaps the solution is to craft a few house rules about NT Default and other lawsets, in a similar fashion to Asimov. They shouldn't restrict player freedom too much, but they should state that those lawsets are not a free grief ticket.

**Pull request merged**